_posts/2025-02-18-JabRef5-15.md

```diff
@@ -0,0 +1,52 @@
+---
+title: JabRef's new GUI for Bibliography Consistency Check
+author: [Priyanshu Gupta](https://github.com/priyanshu16095)
```

Comment:
You first need to add your name to the author's file; otherwise, it's not shown correctly
https://github.com/JabRef/blog.jabref.org/blob/main/_data/authors.yml